lib/config.js

```diff
@@ -73,7 +73,8 @@ var DEFAULTS = {
   transactionMaxSpans: 500,
   transactionSampleRate: 1.0,
   usePathAsTransactionName: false,
-  verifyServerCert: true
+  verifyServerCert: true,
+  useElasticTraceparentHeader: true
```

Comment:
We used to sort these lists alphabetically - probably best to keep doing that :)